templates/admin/notice.tmpl

```diff
@@ -35,31 +35,33 @@
 					{{end}}
 				</tbody>
 				<tfoot class="full-width">
-					<tr>
-						<th></th>
-						<th colspan="5">
-							<div class="ui right">
-								<a class="ui red small button" href="{{AppSubUrl}}/admin/notices/empty">{{.i18n.Tr "admin.notices.delete_all"}}</a>
-							</div>
-							<div class="ui floating upward dropdown small button">
-								<span class="text">{{.i18n.Tr "admin.notices.actions"}}</span>
-								<div class="menu">
-									<div class="item select action" data-action="select-all">
-										{{.i18n.Tr "admin.notices.select_all"}}
-									</div>
-									<div class="item select action" data-action="deselect-all">
-										{{.i18n.Tr "admin.notices.deselect_all"}}
-									</div>
-									<div class="item select action" data-action="inverse">
-										{{.i18n.Tr "admin.notices.inverse_selection"}}
+					{{ if gt (len .Notices) 0 }}
```

Comment:
Also I think this `{{ if ... }}` should be before `<tfoot class="full-width">` and `{{ end }}` after `</tfoot>`

Reply:
Added that already and rebased